src/internal/plugins/controllers/drawers.ts

```diff
@@ -24,10 +24,13 @@ export interface DrawerConfig {
   unmountContent: (container: HTMLElement) => void;
 }
 
+export type UpdateDrawerConfig = Partial<Omit<DrawerConfig, 'onResize' | 'mountContent' | 'unmountContent'>>;
```

Comment:
Open for discussion here, but I think it should be an allow-list for known supported properties. For example we do not want `defaultActive`, `defaultSize` or `iconSvg` to be modifiable.

Can we change this to `Pick` only for properties we definitely want to be dynamic?

Reply:
Sounds good, I was kind of iffy on this. That's good cause out.

Reply:
updated, keeping `defaultSize`. it would be useful in some cases.